masterlist.yaml

```diff
@@ -2359,6 +2307,21 @@ plugins:
         util: 'SSEEdit v3.2.1'
   - name: 'RealisticWaterTwo.esp'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/2182/' ]
+    group: *rwtGroup
+    tag:
+      - C.Water
+      - Graphics
+      - Sound
+    clean:
+      - crc: 0xB2812DAF # v1.4.4
+        util: 'SSEEdit v3.2.1'
+      - crc: 0xCF76C1B9 # v1.5.2
+        util: 'SSEEdit v3.2.1'
+      - crc: 0x2BDAE302 # v1.5.3
+        util: 'SSEEdit v3.2.1'
+  - name: 'RealisticWaterTwo.*\.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/2182/' ]
+    group: *rwtGroup
     after:
```

Comment:
Swapped sort after rules to regex. On the off chance that an optional plugin does not have RWT as a master.